sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/StatusData.java

```diff
@@ -38,8 +39,8 @@ static StatusData error() {
    * Returns a {@link StatusData} with the given {@code code} and {@code description}. If {@code
    * description} is {@code null}, the returned {@link StatusData} does not have a description.
    */
-  static StatusData create(StatusCode code, String description) {
-    return ImmutableStatusData.create(code, description);
+  static StatusData create(StatusCode code, @Nullable String description) {
+    return ImmutableStatusData.create(code, description != null ? description : "");
```

Comment:
We use empty string for the description for our no-description singletons so figure this was intended. Currently the code seems buggy since it stores a null description which can be accessed by non-null getDescription